.github/workflows/static_analysis.yaml

```diff
@@ -47,18 +52,18 @@ jobs:
         run: |
           set -eu
 
-          DIFF="$(git diff --name-only origin/${{ github.base_ref }} ${{ github.sha }})"
-          echo "Diff between ${{ github.base_ref }} and ${{ github.sha }}"
+          DIFF="$(git diff --name-only origin/${{ github.base_ref }} origin/${{ github.head_ref }} -- '*.tf')"
+          echo "Diff between ${{ github.base_ref }} and ${{ github.head_ref }}"
           echo "files=$(echo \"$DIFF\" | xargs echo)" >> $GITHUB_OUTPUT
 
-      - name: Cache pre-commit
-        uses: actions/cache@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 # v4.0.2
-        with:
-          path: ~/.cache/pre-commit
-          key: pre-commit-3|${{ hashFiles('.pre-commit-config.yaml') }}
+      - name: Fix Alpine Container Image
```

Comment:
why it's not cached anymore?

Reply:
I didn't notice any performance boost, it seemed quite useless